s4e-backend/src/main/java/pl/cyfronet/s4e/controller/response/BasicProductCategoryResponse.java

```diff
@@ -1,6 +1,11 @@
 package pl.cyfronet.s4e.controller.response;
 
+import org.springframework.beans.factory.annotation.Value;
+
 public interface BasicProductCategoryResponse {
     Long getId();
     String getLabel();
+
+    @Value("#{@productCategoryService.getIconUrlBy(target.iconName)}")
+    String getUrl();
```

Comment:
I argue, that this field should be named `iconPath` and not `url`.
The value returned by `productCategoryService.getIconUrlBy` is actually something like `categories/icon_sth.svg`, which definitely isn't a [URL](https://en.wikipedia.org/wiki/URL). Even when predending the value with bucket name and getting `static/categories/icon_sth.svg` doesn't fulfill the requirements for a URL, even though clicking such link in context of a browser page would lead you to the resource.
IconPath, on the other hand serves two ends: describes clearly that it is only a path, not a full URL, and describes the content stored at this path, an icon.

@michal-szostak, @kmarszalek? What are your opinions on this, as we cannot get with @danielkryska to understanding.